scripts/demo-projects.sh

```diff
@@ -12,6 +12,8 @@ if [ "$REACT_NATIVE_COMPAT_TEST" != "true" ]; then
   node scripts/change_react_native_version.js "detox" ${REACT_NATIVE_VERSION} "devDependencies"
 fi
 
+# Update the dependencies of the test project because bootstrap will run and fail
+node scripts/change_react_native_version.js "detox/test" ${REACT_NATIVE_VERSION} "dependencies"
```

Comment:
Not sure why this is needed. This is a script for the demo projects 🤔

Reply:
Because lerna runs post install for all projects. There are some special hacks for making it work on rn 71 or rn 73. So if you don't update the versions correctly to all projects it will crash on pod install